bin/configs/java-restclient-nullable-array.yaml

```diff
@@ -0,0 +1,8 @@
+generatorName: java
+outputDir: samples/client/petstore/java/restclient-nullable-arrays
```

Comment:
thanks for the PR.

can you please add the new folder to  .github/workflows/samples-java-client-jdk11.yaml so that the CI can test it moving forward?

Reply:
I added the directories to the Samples Java Client JDK 17. The 17 one because the needed Spring versions minimum Java version is 17.